test/libs/api_pipeline_test.py

```diff
@@ -69,31 +62,27 @@ def test_build_api_output_for_intervention(nyc_fips, nyc_model_output_path, tmp_
     assert sorted(output_paths) == sorted(expected_outputs)
 
 
-def test_latest_values_no_unknown_fips(tmp_path):
-    unknown_fips = "69999"
-    us_latest = combined_datasets.load_us_latest_dataset()
-    us_timeseries = combined_datasets.load_us_timeseries_dataset()
-    latest = us_latest.get_subset(fips=unknown_fips)
-    timeseries = us_timeseries.get_subset(fips=unknown_fips)
-    all_timeseries_api = api_pipeline.run_on_all_fips_for_intervention(
-        latest, timeseries, Intervention.OBSERVED_INTERVENTION, tmp_path
+def test_output_no_timeseries_rows(nyc_region, tmp_path):
+    regional_input = api_pipeline.RegionalInput.from_region_and_intervention(
+        nyc_region, Intervention.OBSERVED_INTERVENTION, tmp_path
     )
-    assert not all_timeseries_api
 
-
-def test_output_no_timeseries_rows(nyc_fips, tmp_path):
-    us_latest = combined_datasets.load_us_latest_dataset()
-    us_timeseries = combined_datasets.load_us_timeseries_dataset()
-    latest = us_latest.get_subset(fips=nyc_fips)
-    timeseries = us_timeseries.get_subset(fips=nyc_fips)
-
-    # Clearing out all rows, testing that a region with no rows still has an API output.
-    timeseries.data = timeseries.data.loc[timeseries.data.fips.isna()]
-
-    assert timeseries.empty
-
-    all_timeseries_api = api_pipeline.run_on_all_fips_for_intervention(
-        latest, timeseries, Intervention.OBSERVED_INTERVENTION, tmp_path
+    # Creating a new regional input with an empty timeseries dataset
+    timeseries = regional_input.timeseries
+    timeseries_data = timeseries.data.loc[timeseries.data.fips.isna()]
+    regional_data = combined_datasets.RegionalData(
+        regional_input._combined_data.region,
```

Comment:
use `nyc_region` or `regional_input.region` instead of single underscore attribute